.github/drafts/16506/16506_category_slider_outline.md

```diff
@@ -0,0 +1,62 @@
+*** Category Slider ***
+
+The Category Slider widget allows users to choose from a fixed set of incremental values.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Surveys, i.e. Strongly agree/ agree/ neutral/ disagree/ strongly disagree
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
+
+These properties allow you to edit the Category Slider widget. The following table lists all the widget's unique properties; if you do not see a certain widget property here, it may be listed under [Common Widget Properties]().
```

Comment:
Okay! That makes more sense. I'll add the common properties to the table and hyperlink the names, so that a click will take the user to the Widgets landing page with the Common Properties section.

Reply:
Could you please update the doc so that we can close the outline review?